packages/ansible-lint/ansible-lint.spec

```diff
@@ -1,9 +1,12 @@
+%global python3_pkgversion 3.11
+%global __python3 /usr/bin/python3.11
+%{?python_disable_dependency_generator}
```

Comment:
Yes, when we don't disable this on EL9 the dependency generation gets a little complicated since some of our packages don't generate the same metadata, it creates a repoclosure problem.